pythonosc/osc_server.py

```diff
@@ -41,10 +41,10 @@ def _is_valid_request(request: List[bytes]) -> bool:
             or osc_message.OscMessage.dgram_is_message(data))
 
 
-class OSCUDPServer(socketserver.UDPServer):
+class OSCUDPServer(socketserver.UDPServer, is_listen=True):
     """Superclass for different flavors of OSC UDP servers"""
 
-    def __init__(self, server_address: Tuple[str, int], dispatcher: Dispatcher) -> None:
+    def __init__(self, server_address: Tuple[str, int], dispatcher: Dispatcher, is_listen) -> None:
```

Comment:
you're not using `is_listen` anywhere, did you forget to pass it down to where it's needed?

Reply:
ahh  you are right. I just changed it at the web-IDE and didn't noticed, that i add the parameter at the wrong position.
Now it should make sense and pass the bool to the constructor of the TCP class of the python socket lib.